packages/cloud_functions/lib/cloud_functions.dart

```diff
@@ -22,7 +22,7 @@ class CloudFunctionsException implements Exception {
 class CloudFunctions {
   CloudFunctions({FirebaseApp app, String region})
       : _app = app ?? FirebaseApp.instance,
-        _region = region;
+        _region = region ?? "us-central1";
```

Comment:
I feel like defaulting to a certain value behind the scene might be hard to debug for users that does not enable the "us-central1" region in their application. Do google cloud always enable "us-central1"? 
Maybe `assert (region != null) is better.
@collinjackson

Reply:
I agree, but apparently the Cloud Functions docs say that they run in the us-central1 region by default. [<Link>](https://firebase.google.com/docs/functions/locations)